packages/eslint-config-airbnb/rules/react.js

```diff
@@ -265,12 +265,18 @@ module.exports = {
           'getInitialState',
           'state',
           'getChildContext',
+          'getDerivedStateFromProps',
           'componentWillMount',
+          'UNSAFE_componentWillMount',
           'componentDidMount',
           'componentWillReceiveProps',
+          'UNSAFE_componentWillReceiveProps',
           'shouldComponentUpdate',
           'componentWillUpdate',
+          'UNSAFE_componentWillUpdate',
+          'getSnapshotBeforeUpdate',
           'componentDidUpdate',
+          'componentDidCatch',
```

Comment:
Now it's present twice: here and two lines below. Is it intended, or just a side effect of two parallel PRs?

Reply:
oops, it's a side effect. i'll clean it up prior to release (or, a PR is helpful)

Reply:
Sure https://github.com/airbnb/javascript/pull/2108